docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,118 @@
+  * Decreasing the possibility in creating new bugs in existing projects because no code churn is required in those projects.
+
+* <a name="support-pcl"></a>[**Keep existing projects and create Portable Class Libraries (PCLs) targeting .NET Core**][option-pcl]
+
+  *What this is good for:*
+  * Referencing your .NET Core libraries in desktop and/or web projects targeting the full .NET Framework in the same solution.
+  * Supporting modifications in the project build or load process. These modifications could be the inclusion of MSBuild Tasks and Targets in your *.csproj file.
+
+  *Unsupported scenarios:*
+  * Does not allow you to write code for a specific .NET Framework version because compiler directives are not supported.
+
+## Example
+
+Consider the repository below:
+
+![][example-initial-project]
+
+[**Source Code**][example-initial-project-code]
+
+There are several different ways to add support for .NET Core for this repository depending on the constraints and complexity of existing projects which are described below.
+
+### Replace Existing Projects with a Multi-targeted .NET Core Project (xproj)
+
+The repository can be reorganized so that any existing *.csproj files are removed and a single *.xproj is created that targets multiple frameworks.  This is a great option because a single project is able to compile for different frameworks.  It also has the power to handle different compilation options, dependencies, etc. per targeted framework.
+
+![][example-xproj]
+
+[**Source Code**][example-xproj-code]
+
+Changes to note are:
+* Addition of `global.json`
+* Replacement of `packages.config` and `*.csproj` with `project.json` and `*.xproj`
+* Changes in the [Car's project.json][example-xproj-projectjson] and its [test project][example-xproj-projectjson-test] to support building for the existing .NET Framework as well as others
+
+### Create a Portable Class Library (PCL) to target .NET Core
+
+If existing projects contain complex build operations or properties in their *.csproj, it may be easier to create a PCL.
+
+![][example-pcl]
+
+[**Source Code**][example-pcl-code]
+
+Changes to note are:
+*  Renaming `project.json` to `{project-name}.project.json`
+    * This prevents potential conflict in Visual Studio when trying to restore packages for the libraries in the same directory. For more information, see the [NuGet FAQ](https://docs.nuget.org/consume/nuget-faq#working-with-packages) under "_I have multiple projects in the same folder, how can I use separate packages.config or project.json files for each project?_".
+    *  **Alternative**: Create the PCL in another folder and reference the original source code to avoid this issue.  Placing the PCL in another folder has an added benefit that users who do not have Visual Studio 2015 can still work on the older projects without loading the new solution.
```

Comment:
It would conflict because the load order for Nuget configurations is:
1. project.json
2. {projectname}.project.json
3  packages.config
4. packages.{projectname}.config

So trying to open the old solution while keeping the packages.config in the old folder, it'll always try to load an existing project.json.